docs/hackathons/eth-global-brussels-hackathon.md

```diff
@@ -0,0 +1,500 @@
+---
+sidebar_position: 1
+---
+
+# 🇧🇪 Chronicle @ETHGlobal Brussels 
+
+🛠️💻🛠️ Hello builders and welcome to ETHGlobal Brussels Chronicle bounties page! 
+
+ This document serves as a guide for integrating Chronicle Oracles into your hackathon projects. 
+
+## 💰 Bounties: 10K split across two tracks
+### Track 1: DeFi Track (Integrate Chronicle Oracles into one of the following DeFi applications: Borrowing & Lending dApps, Decentralized Exchanges (DEX), Liquidity Mining, Prediction Markets, or Stablecoins)
+
+- 🥇 1st: $2650
+- 🥈 2nd: $1500
+- 🥉 3rd: $850
+
+### Track 2: Most Innovative Chronicle Oracle Integration
+
+- 🥇 1st: $2650
+- 🥈 2nd: $1500
+- 🥉 3rd: $850
+
+The prize will go to the top three teams with the most innovative integration of Scribe, Chronicle's new Schnorr Oracle. You can deploy your contracts on any of the following chains:
+
+- Ethereum Sepolia
+- Arbitrum Sepolia
+- Base Sepolia
+- Polygon zkEVM Sepolia
+- Gnosis Chain Mainnet
+- Scroll Sepolia
+- zkSync Sepolia 
+- Polygon zkEVM Sepolia
```

Comment:
Polygon zkEVM mentioned 2 times

Reply:
Thank you, Pascal. I wrote to Alex this morning to try to squeeze in in the engineering pipeline the deployments for the SelfKisser on Arbitrum, Base, and Optimism Testnet.